src/ui/Quickview/Quickview.ts

```diff
@@ -270,17 +272,23 @@ export class Quickview extends Component {
     return icon;
   }
 
-  private buildTootip(icon: HTMLElement) {
-    const tooltip = $$('div', { className: 'coveo-caption-for-icon', tabindex: 0 }, 'Quickview'.toLocaleString()).el;
-    const arrow = $$('div').el;
-    tooltip.appendChild(arrow);
+  private buildCaption() {
+    return $$('div', { className: 'coveo-caption-for-icon', tabindex: 0 }, 'Quickview'.toLocaleString()).el;
+  }
 
-    this.buildPopper(icon, tooltip, arrow);
-    return tooltip;
+  private buildTooltipIfNotInCardLayout(icon: HTMLElement, caption: HTMLElement) {
+    const layout = this.queryStateModel.get(QueryStateModel.attributesEnum.layout);
```

Comment:
It would be nice to extract the layout check into it's own method. e.g.

```
private get resultsAreInCardLayout() {
    return this.queryStateModel.get(QueryStateModel.attributesEnum.layout) == 'card'
}
```

and then the usage,
```
if (this.resultsAreInCardLayout) {
    return;
}
```